src/ontology/oba-edit.obo

```diff
@@ -1290,37 +1285,11 @@ id: OBA:1001088
 name: obsolete race
 is_obsolete: true
 
-
-[Typedef]
-id: has_part ! has part
-xref: BFO:0000051
-is_transitive: true
-inverse_of: part_of ! part_of
-
 [Typedef]
-id: occurs_in ! occurs in
-xref: BFO:0000066
-transitive_over: part_of ! part_of
-
-[Typedef]
-id: part_of ! part of
-xref: BFO:0000050
-is_transitive: true
-
-[Typedef]
-id: inheres_in_part_of ! inheres in part -of
-xref: RO:0002314 ! inheres in part of
-holds_over_chain: inheres_in part_of
-
-[Typedef]
-id: inheres_in ! inheres in
-is_a: inheres_in_part_of
-xref: RO:0000052 ! inheres in
-
-[Typedef]
-id: http://purl.org/pav/providedBy ! provided by
+id: http://purl.org/dc/elements/1.1/contributor ! contributor
```

Comment:
add `is_metadata_tag: true`

Reply:
That is there, just a bit.. lower down in the github diff :P